presto-main/src/main/java/com/facebook/presto/security/AccessControlManager.java

```diff
@@ -495,6 +497,33 @@ public void checkCanSetCatalogSessionProperty(TransactionId transactionId, Ident
         }
     }
 
+    @Override
+    public void checkCanCreateRole(TransactionId transactionId, Identity identity, String role, Optional<PrestoPrincipal> grantor, String catalogName)
+    {
+        requireNonNull(identity, "identity is null");
```

Comment:
It is done in all the methods in this class. I just wanted to keep it consistent.